specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2657,7 +2765,10 @@
       ],
       "description": "The aggregation.",
       "discriminator": "kind",
-      "type": "object"
+      "type": "object",
+      "required": [
+        "kind"
+      ]
```

Comment:
I suppose this is already being enforced at server level and even discriminator pattern may not work. hope that is the case.
otherwise, adding new required property is a breaking change.

Reply:
I added these because a test wouldn't pass, there was an error saying a discriminator must be required.
It is enforced in the server level as well.
What do you mean by "even discriminator pattern may not work" ?